.gitignore

```diff
@@ -29,3 +29,6 @@ npm_licenses.tar.bz2
 /.build
 
 /**/node_modules
+
+/.idea
```

Comment:
We do not change .gitignore, please use your local gitignore to ignore your IDE.

Reply:
Reverted